### PR TITLE
feat: complete hooks tutorial component

### PR DESCRIPTION
**Pull request by curious-curie**
## What does this PR do?

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Code Style Update (formatting, renaming)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe):

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: frontend/.eslintrc.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "prettier/prettier": ["error"],
     "comma-dangle": ["error", "never"],
     "import/no-unresolved": "off",
+    "no-use-before-define": "off",
```
Comment (ssm0318) [inline frontend/.eslintrc.json:24]: 마음에 안 들어요

File: frontend/src/constants.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export const mockQuestions = [
+  {
+    id: 0,
+    question: '어디서 마시는 커피를 가장 좋아하는가?'
```
Comment (ssm0318) [inline frontend/src/constants.js:4]: 잘했어요

**Review by ssm0318 — CHANGES_REQUESTED** [review]
굿잡~~~

File: frontend/src/constants.js (hunk 2 above)
Comment (curious-curie) [reply frontend/src/constants.js:4]: thanks

**Review by koyrkr — APPROVED** [review]

**Pull request merged**